app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+
```

Comment:
We need a new session action (log in form?) here as well. Maybe we can clean up the p and puts debugging statements.